deploy/stacks/pipeline.py

```diff
@@ -634,13 +686,13 @@ def set_stacks_updater_stage(
                     'aws sts get-caller-identity --profile buildprofile',
                     f"export cluster_name=$(aws ssm get-parameter --name /dataall/{target_env['envname']}/ecs/cluster/name --profile buildprofile --output text --query 'Parameter.Value')",
                     f"export private_subnets=$(aws ssm get-parameter --name /dataall/{target_env['envname']}/ecs/private_subnets --profile buildprofile --output text --query 'Parameter.Value')",
-                    f"export security_groups=$(aws ssm get-parameter --name /dataall/{target_env['envname']}/ecs/security_groups --profile buildprofile --output text --query 'Parameter.Value')",
+                    f"export security_groups=$(aws ssm get-parameter --name /dataall/{target_env['envname']}/ecs/cdkproxy_security_groups --profile buildprofile --output text --query 'Parameter.Value')",
```

Comment:
I think this name does not exist in container.py we define `f'/dataall/{envname}/ecs/security_groups',`

Reply:
Good catch - will fix